### DISCUSSION TITLE
Option to disable Recaptcha for the login page

### DISCUSSION THREAD

**Discussion by SpaceGameDev568** [pelican-dev/panel#285] [Ideas]
Can there be an option to disable using ReCaptcha for the login page on the panel? I don't want/need it, and I'd prefer that my panel wasn't dependent on a Google service.

**Comment by Boy132** [comment]
There already is an option for that. Set `RECAPTCHA_ENABLED` to false in your .env file.

**Reply by SpaceGameDev568** [reply]
Sweet, thanks!
